### PR TITLE
Update oudated documentation around instantiating a ClientConfig and …

### PR DESCRIPTION
**Pull request by anwar6953**
…setting its AccessToken.

### CODE REVIEW

**Comment by jwang47** [comment]
LGTM

File: cdap-docs/reference-manual/source/java-client-api.rst [hunk 1]
```diff
@@ -72,7 +72,9 @@ instance that you wish to interact with.
 In a non-secure (default) CDAP instance, instantiate as follows::
 
   // Interact with the CDAP instance located at example.com, port 10000
-  ClientConfig clientConfig = new ClientConfig("example.com", 10000);
+  ClientConfig clientConfig = ClientConfig.builder()
+    .setConnectionConfig(new ConnectionConfig("example.com", 10000, false))
+    .build()
```
Comment (awholegunch) [inline cdap-docs/reference-manual/source/java-client-api.rst]: shouldn't this be
```
.build();

**Comment by awholegunch** [comment]
One comment, otherwise LGTM. Fix and merge...

**Comment by anwar6953** [comment]
Fixed and merged.

**Pull request merged**